
Showing logs for build [32mmeta-entando-k8s-entando-de-app-8vwxl-1[0m stage [32mmeta-pipeline[0m and container [32mstep-credential-initializer-hxxlm[0m
{"level":"warn","ts":1603893532.1435578,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1603893532.1526139,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mmeta-entando-k8s-entando-de-app-8vwxl-1[0m stage [32mmeta-pipeline[0m and container [32mstep-working-dir-initializer-gjnz2[0m
{"level":"warn","ts":1603893534.3082895,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1603893534.3252645,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully e**cuted command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mmeta-entando-k8s-entando-de-app-8vwxl-1[0m stage [32mmeta-pipeline[0m and container [32mstep-place-tools[0m

Showing logs for build [32mmeta-entando-k8s-entando-de-app-8vwxl-1[0m stage [32mmeta-pipeline[0m and container [32mstep-git-source-meta-entando-k8s-entando-de-app-vts6s[0m
{"level":"warn","ts":1603893544.1906877,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1603893546.4496644,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/entando-k8s/entando-de-app.git @ 4cd4ecd3ae378c8edf4c1a314f0ac631025618e3 in path /workspace/source"}

Showing logs for build [32mmeta-entando-k8s-entando-de-app-8vwxl-1[0m stage [32mmeta-pipeline[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:4cd4ecd3ae378c8edf4c1a314f0ac631025618e3,121:ab249f6e224ad496def5155d9cac61bcca7aac07
DEBUG: ran git fetch --unshallow origin ab249f6e224ad496def5155d9cac61bcca7aac07: 4cd4ecd3ae378c8edf4c1a314f0ac631025618e3: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard 4cd4ecd3ae378c8edf4c1a314f0ac631025618e3 in 
DEBUG: ran clean --force -d . in 
error: error during merge: merging ab249f6e224ad496def5155d9cac61bcca7aac07 into master: git output: Auto-merging pom.xml
CONFLICT (content): Merge conflict in pom.xml
Automatic merge failed; fix conflicts and then commit the result.: failed to run 'git merge ab249f6e224ad496def5155d9cac61bcca7aac07' command in directory '', output: 'Auto-merging pom.xml
CONFLICT (content): Merge conflict in pom.xml
Automatic merge failed; fix conflicts and then commit the result.'
[31m
Pipeline failed on stage 'meta-pipeline' : container 'step-git-merge'. The execution of the pipeline has stopped.[0m
